conf/authdata.json

```diff
@@ -15,7 +15,7 @@
             "arn": "aws::iam:123456789013:bart",
             "keys": [{
                 "access": "USERBARTFUNACCESSKEY",
-                "secret": "verySecretKey1"
+                "secret": "verySecretKeyUser1"
```

Comment:
This would mean that if any user in open-source was using this credential, it will stop working. It will be nice to add this to the release notes when you push a new Docker open-source docker image.